sdk/spring/azure-spring-boot-test-keyvault/src/test/java/com/microsoft/azure/test/keyvault/KeyVaultIT.java

```diff
@@ -164,7 +155,13 @@ public void keyVaultWithVirtualMachineMSI() throws Exception {
             AZURE_KEYVAULT_URI,
             "app.jar"));
 
-        vm.runCommand(new RunCommandInput().withCommandId("RunShellScript").withScript(commands));
+        try (SSHShell sshShell = SSHShell.open(host, 22, VM_USER_USERNAME, VM_USER_PASSWORD);
```

Comment:
Why make such changes here?

Reply:
Because VirtualMachine.runCommands() cannot find the java environment when executing commands in the virtual machine.

Reply:
And moving the command will help with the Java environment?

Reply:
No, I just changed the way of executing commands from VirtualMachine.runCommands() to SSHShell.runCommands(). Command still needs to be executed.

Reply:
Okay. Let's try fixing this Java environment issue then.

Reply:
Sorry, after testing, if the virtual machine is created through test-resources.json, this problem does not occur, but if it is a virtual machine created by myself, there will be a problem of not being able to find the java environment. Maybe some configuration is missing in the virtual machine I created?

Reply:
`test-resources.json` have [a step to install java](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/spring/azure-spring-boot-test-keyvault/test-resources.json#L214).